observation_portal/common/mixins.py

```diff
@@ -71,3 +71,43 @@ def to_internal_value(self, data):
                     except (ValueError, TypeError):
                         pass
         return data
+
+class GetSerializerMixin(object):
```

Comment:
I created this mixin to be used by the `APIView` classes so that we could define a `serializer_class` attribute and DRF autodocs would pick it up as the serializer for that view. The base APIView does not include this functionality by default.